fingerprint/src/main/java/com/fingerprintjs/android/fingerprint/tools/logs/Messages.kt

```diff
@@ -0,0 +1,8 @@
+package com.fingerprintjs.android.fingerprint.tools.logs
+
+internal fun Logger.ePleaseReport(throwable: Throwable?) {
+    e(
+        msg = "Unexpected error occurred. Feel free to create an issue on Github.",
```

Comment:
Let's mark this message with "... on Github repository of the fingerprintjs-android library." for the cases when all the stacktrace will be obfuscated.

Reply:
Nice catch, fixed.